modules/geo-layers/src/mvt-layer/mvt-layer.js

```diff
@@ -137,7 +139,13 @@ export default class MVTLayer extends TileLayer {
       props.extensions = [...(props.extensions || []), new ClipExtension()];
     }
 
-    return super.renderSubLayers(props);
+    const subLayers = super.renderSubLayers(props);
+
+    if (this.props.binary && !(subLayers instanceof GeoJsonLayer)) {
+      throw new Error('renderSubLayers() must return GeoJsonLayer when using binary:true');
```

Comment:
This will hard crash the layer if e.g. a different copy of `@deck.gl/layers` is imported. Consider using `log.warn` instead.

Reply:
Switched to log.warn